components/Chips/src/ColorThemer/MDCChipViewColorThemer.h

```diff
@@ -34,12 +34,13 @@
                       toChipView:(nonnull MDCChipView *)chipView;
 
 /**
- Applies a color scheme's properties to a stroked MDCChipView.
+ Applies a color scheme's properties to a stroked @c MDCChipView.
 
  @param colorScheme The color scheme to apply to the component instance.
- @param strokedChipView A component instance to which the color scheme should be applied.
+ @param chipView @c A component instance to which the color scheme should be applied.
  */
-+ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
```

Comment:
I believe this API already landed unfortunately, so we'll have to mark it as to-be-deprecated.

Reply:
It has landed, but it seems no one is using it yet. With that in mind can we sneak it out without deprecating?

Reply:
You can mark it as deprecated. Once we cut the next release you can delete it.